src/apps/chifra/pkg/types/types_transaction.go

```diff
@@ -172,4 +233,9 @@ func (s *SimpleTransaction) Model(showHidden bool, format string, extraOptions m
 }
```

Comment:
I'm just noting here that the "rendering" code can get quite complicated given all the options. At some point (not now), we may revisit how some of these things work. For example, the many alternative "formats" might be "raised up" into the calling code so that this "type-related" code just does as instructed. We talked about this.